src/printers/logs.ts

```diff
@@ -0,0 +1,19 @@
+import { LogList, LogApiResource } from '@twilio-labs/serverless-api';
+import { LogsConfig } from '../config/logs';
+import { logger } from '../utils/logger';
+
+export function printLogs(
+  result: LogApiResource[],
+  config: LogsConfig,
+  outputFormat?: string
+) {
+  result.forEach(log => printLog(log, outputFormat));
+}
+
+export function printLog(log: LogApiResource, outputFormat?: string) {
+  if (outputFormat === 'json') {
+    logger.info(JSON.stringify(log));
```

Comment:
`logger.info` will log to `stderr` not to `stdout`. You should instead use `writeOutput` from the `../utils/output` module.

Reply:
Thanks! Thought I was being good using `logger.info`. Updated.